ajax/libs/jodit/package.json

```diff
@@ -16,12 +16,26 @@
     "type": "git",
     "url": "https://github.com/xdan/jodit.git"
   },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xdan/jodit.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "jodit.min.js",
+          "jodit.min.css"
+        ]
+      }
+    ]
+  },
   "npmName": "jodit",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "build",
       "files": [
-        "*jodit*.+(css|js|map)"
+        "jodit.min.js",
+        "jodit.min.css"
```

Comment:
Let's modify from A to B.
- A
```js
        "jodit.min.js",
        "jodit.min.css"
```

- B
```js
        "**/*"
```
this pattern will get all the files which under `build` folder.
Thank you.